browser-test/src/applicant/applicant_question_translation.test.ts

```diff
@@ -0,0 +1,182 @@
+import {test, expect} from '../support/civiform_fixtures'
+import {
+  loginAsAdmin,
+  logout,
+  selectApplicantLanguage,
+  validateScreenshot,
+  validateToastMessage,
+} from '../support'
+
+test.describe(
+  'Admin can manage translations',
+  {tag: ['@uses-fixtures']},
+  () => {
+    test('creates a question and adds translations', async ({
+      page,
+      adminPrograms,
+      adminQuestions,
+      adminTranslations,
+      applicantQuestions,
+    }) => {
+      await loginAsAdmin(page)
+
+      // Add a new question to be translated
```

Comment:
suggestion (non-blocking): Use `test.step` for better debugging and to get rid of comments: https://github.com/civiform/civiform/wiki/Testing#steps

Reply:
When I write new tests, I'll use `test.step`. Since this is just a refactor, I'll leave that change out of this PR.